modules/http/pom.xml

```diff
@@ -79,6 +79,18 @@
             <groupId>commons-codec</groupId>
             <artifactId>commons-codec</artifactId>
         </dependency>
+        <dependency>
```

Comment:
I don't think we need one. Remember that the whole http module will be removed at some point before 4.0 release.